src/controllers/users.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { usersService } = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.send(users);
+};
+
+const getById = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = usersService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
```

Comment:
I think here we have also check for incorrect userId type (isNaN(+userId)) and send status 400